discord/state.py

```diff
@@ -281,7 +281,10 @@ def parse_guild_member_remove(self, data):
     def parse_guild_member_update(self, data):
         server = self._get_server(data.get('guild_id'))
         user_id = data['user']['id']
-        member = server.get_member(user_id)
+        try:
+            member = server.get_member(user_id)
+        except AttributeError:
```

Comment:
`server.get_member` does not ever throw any sort of error. It will return `None` if the user_id is not found.